src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/handlers/CcdCallbackHandler.java

```diff
@@ -141,7 +141,8 @@ private String stringJoin(List<String> messages) {
             callback.getCaseDetails().getCaseData().getSscsDocument(),
             callback.getCaseDetails().getCaseData().getSubscriptions(),
             formType,
-            callback.getCaseDetails().getCaseData().getChildMaintenanceNumber()
+            callback.getCaseDetails().getCaseData().getChildMaintenanceNumber(),
```

Comment:
Rasika and I were discussing if we can keep all SSCS2 data in a class with builder and use it